josh-proxy/Cargo.toml

```diff
@@ -37,6 +37,6 @@ serde_json= "1.0"
 serde= "*"
 rust-crypto = "0.2"
 unindent = "*"
-juniper = "*"
+juniper = {version = "*", features =["expose-test-schema"] }
```

Comment:
What does that do?

Reply:
exposes the feature which is now hidden behind this. The tests were not running anymore.